packages/mdc-line-ripple/README.md

```diff
@@ -42,15 +42,15 @@ CSS Class | Description
 --- | ---
 `mdc-line-ripple` | Mandatory
 `mdc-line-ripple--active` | Styles the line ripple as an active line ripple
-`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple. 
+`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple
 
 ### Sass Mixins
 
 ### `MDCLineRipple`
 
-`activate() => void` | Proxies to the foundations `activate()` method.
-`deactivate() => void` |  Proxies to the foundations `deactivate()` method.
-`setRippleCenter(number: xCoordinate) => void` | Proxies to the foundations `setRippleCenter(number: xCoordinate)` method.
+`activate() => void` | Proxies to the foundation's `activate()` method
```

Comment:
Is this supposed to be a table? I think you need the `--- | ---` above

Reply:
You, sir, are very correct, and wowzers this sure looks very broken right now. Good catch.